requirements/README.md

```diff
@@ -0,0 +1,122 @@
+# Requirements
+DeepChem is a python deep-learning library for molecular machine learning. During its installation, it requires various prerequisite libraries to be installed. The various `.yml` files in the `requirements` folder contain the different libraries that are required based on the installer. The different folders are used to show the dependencies of the DeepChem library on the respective deep-learning backends.
+
+The files present are:
+1. `env_common.yml` - contains libraries related to scientific applications that are common to all operating systems.
+2. `env_dqc.yml` - contains libraries specific for using Quantum Chemistry packages through the [Differentiable Quantum Chemistry package](https://github.com/diffqc/dqc).
+3. `env_mac.yml` - contains dependencies specific to macOS.
+4. `env_ubuntu.yml` - contains dependencies specific to Ubuntu.
+5. `env_test.yml` - contains libraries related to testing code and models.
+
+The folders present are:
```

Comment:
we don't need here what jax or pytorch or tensorflow here is. The below three points can be compressed into a single line, something like the three subfolder contains cpu and gpu deep learning backend requirements of deepchem.